thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -104,12 +104,42 @@ export default Ember.Component.extend({
     'entities',
     'aggregates',
     function () {
-      const { entities, aggregates } = this.getProperties('entities', 'aggregates');
-      return filterPrefix(Object.keys(entities), ['thirdeye:metric:'])
-        .reduce((agg, urn) => {
-          agg[urn] = aggregates[toCurrentUrn(urn)] / aggregates[toBaselineUrn(urn)] - 1;
-          return agg;
-        }, {});
+      const { entities, aggregates } = this.getProperties('entities', 'aggregates'); // poll observer
+      return this._changesOffset('baseline');
+    }
+  ),
+
+  changesFormatted: Ember.computed(
+    'changes',
+    function () {
+      const { changes } = this.getProperties('changes');
```

Comment:
Since you're only getting 1 property, you can use `this.get`

Reply:
ack. you know me :-)